demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -0,0 +1,48 @@
+import subprocess
+import os
+from demisto_sdk.commands.common.tools import print_error, print_warning
+
+
+class ReadMeValidator:
+    def __init__(self, file_path):
+        self.file_path = file_path
+        self.is_valid = True
+        self.ready = False
+
+    def is_valid_file(self):
+        """Check whether the readme file is valid or not
+        """
+        is_readme_valid = all([
+            self.is_mdx_file(),
+        ])
+        return is_readme_valid
+
+    def is_mdx_file(self) -> bool:
+        mdx_parse = f'{os.path.dirname(os.path.abspath(__file__))}/../mdx-parse.js'
+        self.are_modules_installed_for_verify()
+        if self.ready:
+            # run the java script mdx parse validator
+            res = subprocess.run(['node', mdx_parse, '-f', self.file_path], text=True, timeout=10,
+                                 capture_output=True)
+            if res.returncode != 0:
+                print_error(f'Failed verifying README.md, Path: {self.file_path}. Error Message is: {res.stderr}')
+                self.is_valid = False
+        return self.is_valid
+
+    def are_modules_installed_for_verify(self):
```

Comment:
we should consider printing the stdout also, if any of dose runs will not return 0.
the check should be after every run.